commands/most-squotable.ts

```diff
@@ -0,0 +1,52 @@
+import * as Discord from 'discord.js';
+import { Message, GuildChannel, TextChannel } from 'discord.js';
+import { Command } from 'squidly/types';
+import { mostFrequentElement } from 'squidly/utils/arrays';
+import { EMBED_COLORS } from 'squidly/constants/colors';
+
+const Keywords = [
+  'squotable',
+  'squoteable',
+  'most_squotable',
+  'most_squoteable',
+  'squotiest',
+  'squoteist',
+  'most_squotiest',
+  'most_squoteist',
+  'top_squoter',
+  'best_squoter',
+  'funniest_member',
+  'funniest_user'
+];
+
+export const MostSquotable: Command = {
+  shouldHandle: (message, { command }) =>
+  Keywords.includes(command),
+  
+  handle: async (message: Message, { tag, command, args }) => {
+    const squotesChannel = message.guild.channels.cache.find((channel: GuildChannel) => {
```

Comment:
Can reduce this to one line
```suggestion
const squotesChannel = message.guild.channels.cache.find(channel => channel.name === ENV.QUOTES_CHANNEL) as TextChannel;
```